baybe/simulation.py

```diff
@@ -607,9 +607,17 @@ def _look_up_target_values(
         #   column ordering, which is not robust. Instead, the callable should return
         #   a dataframe with properly labeled columns.
 
+        # For evaluating the lookup function, we need to filter the task parameter
```

Comment:
This part was responsible for the error that I talked about in my other comment.
The `TaskParameter` was passed to the `lookup` function, which however does not care for the name of the function. Thus, this needs to be removed before passing the values to `lookup`.  Do not destroy the integrity of the `queries` object, a copy is constructed and used for evaluation.
@Scienfitz but specifically @AdrianSosic : Please verify that this fix does not break anything and should be done in this way. During the hackathon, I have created a continuous example which now works using this code (you can find it in the `bug/task_parameter_in_continuous_space` branch in which I very quickly adjusted the `basic_transfer_learning` example)

Reply:
As discussed, I dropped the corresponding change.